lib/controllers/ticket_controller.dart

```diff
@@ -0,0 +1,125 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:surf_flutter_study_jam_2023/models/ticket.dart';
+import 'package:surf_flutter_study_jam_2023/views/ticket_create_view.dart';
+import 'package:flutter/services.dart';
+
+import '../services/fileDownloader.dart';
+
+class TicketController {
+  final downloader = FileDownloader();
+
+  // todo: Попробовать сделать через ValueNotifier
+  // ValueNotifier<String> _myModel = ValueNotifier<String>('initial data');
+
+  final StreamController<List<Ticket>> _ticketStreamController =
+      StreamController<List<Ticket>>.broadcast();
+
+  Stream<List<Ticket>> get tasksStream => _ticketStreamController.stream;
+
+  List<Ticket> _tickets = [];
+
+  Future<void> downloadAllTicket(
+      BuildContext context, List<Ticket>? tickets, Function() callback) async {
+    for (var ticket in tickets!) {
```

Comment:
Using the "!" operator here is a bad practice. It's better to check if it's not null before using it.